ZupIT/ritchie-formulas
# [WIP] - Add horus job to circleci config

# Pull Request

## What I did

Added horus job to circleci config

## How I did it

Changing the `.circleci/.config.yml` file and adding a job according to the Horus documentation (https://horusec.io/).

## How to verify it

The horusec check will happen on the branch whenever a pull request is opened that matches the name pattern specified in the file.

## What kind of change does this PR make


## Description for the changelog

Add horus job to circleci config
